src/resources/postcss/components/_events-bar.pcss

```diff
@@ -159,6 +159,40 @@
 		width: 21px;
 	}
 
+	.tribe-events-c-events-bar__tabs {
+		display: flex;
+	}
+
+	.tribe-events-c-events-bar__tab {
+		border-radius: var(--border-radius-default);
```

Comment:
let's alphabetize all the styles, we don't have a linter yet but the linter will flag these.